internal/tunnel/proto/msg.go

```diff
@@ -79,13 +82,12 @@ type AuthExtra struct {
 type ClientType string
 
 const (
-	Agent   ClientType = "agent"
-	Library ClientType = "library"
+	LibraryOfficialGo ClientType = "library/official/go"
 )
 
 type Fingerprint struct {
-	M []string
-	D []string
+	M []string // mac addresses
+	D []string // disk serial numbers
```

Comment:
I manually stripped those before since, well, they're unused in this library
(we also don't actually populate D in the agent, and iirc never did, and I'd rather not give the impression we did)